rafs/src/metadata/md_v5.rs

```diff
@@ -83,13 +83,18 @@ impl RafsSuper {
         Ok(meta_size as usize)
     }
 
-    pub(crate) fn prefetch_data_v5<F>(&self, r: &mut RafsIoReader, fetcher: F) -> RafsResult<usize>
+    pub(crate) fn prefetch_data_v5<F>(
+        &self,
+        r: &mut RafsIoReader,
+        root_nid: Inode,
```

Comment:
This might be misnamed. V5 has no nid concept

Reply:
Fixed